derived_images/mongodb/README.md

```diff
@@ -0,0 +1,32 @@
+dockerfiles-opensuse-mongodb
+=============================
+
+openSUSE Dockerfile for MongoDB - an open-source document database, and the leading NoSQL database.  
+
+
+To build:
+
+```
+# docker build -t <username>/mongodb .
+```
+
+To run:
+
+```
+# docker run -d -p 27107 <username>/mongodb
```

Comment:
If you map it statically with -p 27107:27107 you don't need to find the port afterwards